beyond-all-reason/Beyond-All-Reason
# Size-Based Mass standardization modoption

This is the Mass Standardization Innitiative come to fruition!

The plan is to have a modoption for now so devs and players can test this stuff out and make tweaks easily. Eventually this is intended to get baked into the unitdefs and then either commented out or more appropriately erased. 

It's designed to be extremely tweakable and hopefully easy to understand so anybody with a mind of balancing can tweak this goodness.

Testing steps:
Enable the modoption in the experimental tab.
Test the units listed in the various tables in alldefs for impulse behavior. Blow stuff up, see how it feels.
